### ISSUE TITLE
Improve approach to getting the data for Ortholog Mapping examples

### ISSUE THREAD

**Issue by cbethell** [AlexsLemonade/refinebio-examples#299]
After some review comments and discussion on the microarray ortholog mapping example notebook update PR #246, there is an outstanding question of how to get the data from HGNC for the Ortholog Mapping example notebooks.

In that PR, we direct users to download the data file directly from the FTP server link: ftp://ftp.ebi.ac.uk/pub/databases/genenames/hcop/

However, as noted in one of @jashapiro's [comments on there](https://github.com/AlexsLemonade/refinebio-examples/pull/246/#discussion_r504672688), clicking this link on a Mac in Safari opens the link in the Finder and this is not the ideal behavior (so we made a note in that PR as a placeholder but would need to find a more robust solution).

Additionally, once a user clicks a file to download, Safari does another less than ideal thing (noted in [another one of Josh's comments](https://github.com/AlexsLemonade/refinebio-examples/pull/246/#discussion_r504692367)) where it automatically decompresses the file so the file name after download would be different across users as those using Safari may have `human_zebrafish_hcop_fifteen_column.txt` while other users will have `human_zebrafish_hcop_fifteen_column.txt.gz` (therefore forcing us to add another note to the notebook in that PR).

To summarize, the general aspects to think about here are how do we handle getting the data while considering:
1. Safari users who cannot access the ftp link
2. Safari users whose downloaded file will be decompressed instead of compressed (.gz)
3. Other users for machines that we have not yet tested (like Windows)

**Comment by cansavvy** [comment]
I suppose for our users we don't necessarily want to have them construct a `wget` command? Or a `download.file()` in R?

**Comment by cansavvy** [comment]
i've tried a download.file() solution. I think its better? Will file a draft PR to see what people think. Also found out that Safari's problems can be circumvented by changing the ftp to http if you just want to see the file itself. (You'll see what I mean in the draft PR).
